usr/sbin/p11sak/p11sak.c

```diff
@@ -4541,8 +4568,11 @@ static CK_RV iterate_objects(const struct p11sak_objtype *objtype,
                              enum p11sak_objclass objclass,
                              CK_RV (*compare_obj)(CK_OBJECT_HANDLE obj1,
                                                   CK_OBJECT_HANDLE obj2,
-                                                  int *result,
-                                                  void *private),
+                                                  int *result
+#if !defined(_AIX)
+                                                  , void *private
```

Comment:
looks a bit strange with the comma at the beginning.....
I guess the alternative would be 

```
                                     CK_RV (*compare_obj)(CK_OBJECT_HANDLE obj1,
                                                          CK_OBJECT_HANDLE obj2,
	                                                  CK_OBJECT_HANDLE obj2,
#if !defined(_AIX)
                                                          int *result,	
                                                          void *private),
#else
                                                          int *result),
#endif                  
```